transcription/transcription.go

```diff
@@ -29,3 +32,25 @@ func msgHeaders(from string, to []string, subject string) string {
 	msgHeaders := []string{fromHeader, toHeader, subjectHeader}
 	return strings.Join(msgHeaders, "\r\n")
 }
+
+// Transcribe a given file using a Sphinx jar. The fileName should be in
+// "name.wav" format and the jarName should be in "name.jar" format.
+func startTranscription(fileName string, jarName string) error {
+
+	cmd := exec.Command("java", "-jar", jarName, fileName)
+	stdout, err := cmd.StdoutPipe()
+	if err != nil {
+		return err
+	}
+	if err := cmd.Start(); err != nil {
+		return err
+	}
+	//Currently copying output to Stdout
+	if _, err := io.Copy(os.Stdout, stdout); err != nil {
+		return err
+	}
+	if err := cmd.Wait(); err != nil {
```

Comment:
Please use https://golang.org/pkg/os/exec/#Cmd.CombinedOutput instead.

1. We want to get stderr too. We could do that with another pipe, but that's just going to make the code more complicated.
2. This is simpler. We always want to block on the command so there is no reason to use pipes as far as I can see.